xbmc/pvr/epg/EpgContainer.cpp

```diff
@@ -312,14 +312,26 @@ bool CPVREpgContainer::PersistAll(unsigned int iMaxTimeslice) const
         CLog::Log(LOGDEBUG, "EPG Container: Persisting events for channel '%s'...",
                   epg->GetChannelData()->ChannelName().c_str());
 
-        bReturn &= epg->Persist(database, true);
+        bReturn &= epg->QueuePersistQuery(database);
+
+        size_t queryCount = database->GetInsertQueriesCount() + database->GetDeleteQueriesCount();
+        if (queryCount > 10000)
+        {
+          CLog::Log(LOGDEBUG, LOGEPG, "EPG Container: committing %d queries in loop.", queryCount);
+          database->CommitDeleteQueries();
+          database->CommitInsertQueries();
+          CLog::Log(LOGDEBUG, LOGEPG, "EPG Container: committed %d queries in loop.", queryCount);
```

Comment:
About new logs can be the `{}` used. And maybe with different strings, on first view I thought both are the same :sweat_smile:. Also not sure if the `queryCount` not changed, to write again.

Maybe?:
```cpp
          CLog::Log(LOGDEBUG, LOGEPG, "EPG Container: Start committing {} queries in loop", queryCount);
          database->CommitDeleteQueries();
          database->CommitInsertQueries();
          CLog::Log(LOGDEBUG, LOGEPG, "EPG Container: End committed queries in loop");
```

Reply:
This id for another day/PR - when I find the time to change whole PVR component to use the new logging syntax.